docs/standard/garbage-collection/large-object-heap.md

```diff
@@ -149,10 +149,9 @@ These performance counters are usually a good first step in investigating perfor
 
    Displays the current size, in bytes, including free space, of the LOH. This counter is updated at the end of a garbage collection, not at each allocation.
 
-A common way to look at performance counters is with Performance Monitor (perfmon.exe). Use “Add Counters” to add the interesting counter for processes that you care about. You can save the performance counter data to a log file, as Figure 4 shows.
+A common way to look at performance counters is with Performance Monitor (perfmon.exe). Use “Add Counters” to add the interesting counter for processes that you care about. You can save the performance counter data to a log file, as the following screenshot shows:
 
-![Figure 4: Adding performance counters.](media/loh/perfcounter.png)\
-Figure 4: The LOH after a generation 2 GC
```

Comment:
let's keep these figure numbers and labels here since the rest of the article still uses these and some of these are still being referenced in the rest of the topic.